src/hooks/locale/locale.js

```diff
@@ -0,0 +1,27 @@
+import { useDataQuery } from '@dhis2/app-runtime'
+import { getLocale } from './selectors'
+
+const fallback = 'en'
+
+const query = {
+    userSettings: {
+        resource: 'userSettings',
+    },
+}
```

Comment:
A GET to `userSettings/keyUiLocale` would be more clean.

Reply:
Ah, yeah that sounds good, I'll change it